lxc: linux Container library

Authors:
Daniel Lezcano <daniel.lezcano@free.fr>

This library is free software; you can redistribute it and/or
modify it under the terms of the GNU Lesser General Public
License as published by the Free Software Foundation; either
version 2.1 of the License, or (at your option) any later version.

This library is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
Lesser General Public License for more details.

ou should have received a copy of the GNU Lesser General Public
License along with this library; if not, write to the Free Software
Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA

Date: May 03 22:17:03 2016

From: 0edbf5181631adf7028980b328a23507ba6e0c53 May 03 22:17:03 2016

Subject: [PATCH] BCOM-1201: lxc class and build time lxc support

	Required for multiuser support using lxc container

Source: COMCAST

Upstream-Status: Pending

Signed-off-by: mselvaraj <Mani_selvaraj@comcast.com>
---

Index: lxc-1.1.2/src/lxc/lxccontainer.c
===================================================================
--- lxc-1.1.2.orig/src/lxc/lxccontainer.c
+++ lxc-1.1.2/src/lxc/lxccontainer.c
@@ -1007,10 +1007,6 @@ static bool create_run_template(struct l
 		}
 
 		if (geteuid() == 0) {
-			if (unshare(CLONE_NEWNS) < 0) {
-				ERROR("error unsharing mounts");
-				exit(1);
-			}
 			if (detect_shared_rootfs()) {
 				if (mount(NULL, "/", NULL, MS_SLAVE|MS_REC, NULL)) {
 					SYSERROR("Failed to make / rslave to run template");
